openapi.yaml

```diff
@@ -12875,6 +12899,8 @@ components:
           $ref: '#/components/schemas/ClusterStateEnum'
         is_deployed:
           type: boolean
+        next_k8s_available_version:
+          type: string
```

Comment:
This field should be in the `ClusterStatusGet` structure (it's not intuitive).
If you look at the status sent as response:
![image](https://github.com/user-attachments/assets/e8a3b899-6e26-4c40-9bfd-6febfbf29e39)

Reply:
Or better, you can remove the `ClusterStatusGet` by `ClusterStatus` and add the field `last_execution_id`, as it is a duplicated struct, don't know why :shrug:

Reply:
Done